Jenkinsfile

```diff
@@ -58,7 +58,7 @@ pipeline {
                     }
                     steps {
                         withDockerRegistry(credentialsId: "${DOCKER_CREDENTIALS_ID}", url: "") {
-                            sh 'sh build-n-publish.sh --image=${DOCKER_IMAGE_NAME} --commit=${GIT_COMMIT} --name=pr-${CHANGE_ID}'
+                            sh 'sh build-n-publish.sh --image=${DOCKER_IMAGE_NAME}${PACKAGE_SUFFIX} --commit=${GIT_COMMIT} --name=pr-${CHANGE_ID}'
```

Comment:
It is [an environment variable](https://github.com/FromDoppler/doppler-jenkins-swarm/pull/5/files#diff-9ca5f821aefab32ac273eddc5c117aa07024d3f26a6b6b0c4a5e7e2c105e880fR28), you can see a description in https://github.com/FromDoppler/doppler-jenkins-swarm/pull/5

TL;DR: I am using it to avoid collisions with the artifacts generated in the new Jenkins. By the moment, it adds the suffix `-new`, but in the future it will be an empty string.